lecture-notes/4-cor.md

```diff
@@ -59,22 +62,28 @@ people if it doesn’t perform the way it’s supposed to. Billions of people re
 on software every day for everything from critical infrastructure to medical
 equipment to silly games. If there's a one in a million chance that your cancer
 radiation therapy machine[^therac25] has a bug that kills you---well, you might
-care. Or if the facial recognition software the state employs has a bug that
+care. Or if the facial recognition software The State employs has a bug that
 lands you in a prison cell, you might care.
 
 [^therac25]: The Therac-25 radiation therapy machine is a case study often used
     in engineering ethics courses. Due to a race condition, the machine
     occasionally dosed people with hundreds of times the radiation they should
     have received, injuring several people and killing several people.
 
-On a less serious note, bugs can lead to revenue loss, or wasting people's
+On a less serious note, bugs can lead to data loss, or revenue loss, or wasting people's
 time. For personal projects, bugs might be inconsequential, like Bob Ross's
 happy little accidents. You won't always be writing code for small projects,
-though.
+though. You might be writing video game software and [accidentally delete
+people's home
```

Comment:
This kind of code is hard to test so I wonder if it is worth writing a "don't just look where the light is" kind of suggestion here... but I don't know what the point is